client/elasticClientCommon.go

```diff
@@ -122,6 +122,10 @@ func extractErrorFromBulkResponse(response *BulkRequestResponse) error {
 	count := 0
 	errorsString := ""
 	for _, item := range response.Items {
+		if item.Index.Status < http.StatusBadRequest {
```

Comment:
why `<` ? if you have a list or status codes returned, please use them

Reply:
Agree, this code is not on the *safe side* as we can easily add bugs when we add new status that are wrongly placed.